core/pom.xml

```diff
@@ -246,6 +246,32 @@
             <artifactId>kafka-schema-registry</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Added Powermock for mocking Static classes

Reply:
should we wrap AdminUtils in a not static class?